src/helpers/test-helpers.js

```diff
@@ -32,7 +32,12 @@ export function axeCheck(component, ignoredRules = []) {
 
   div.innerHTML = '';
 
-  const mountedComponent = mount(component, { attachTo: div });
+  let mountedComponent;
+  if (state) {
+    mountedComponent = mount(component, { attachTo: div }).setState(state);
+  } else {
+    mountedComponent = mount(component, { attachTo: div });
+  }
```

Comment:
What about something like this:

```suggestion
  const mountedComponent = mount(component, attachTo: div});
  return state ? mountedComponent.setState(state) : mountedComponent;
```